waymo_raw_data_to_argoverse.py

```diff
@@ -216,19 +258,89 @@ def dump_pose(city_SE3_egovehicle, timestamp, log_id):
 		'rotation': [w, x, y, z],
 		'translation': [x,y,z]
 	}
-	json_fpath = f'TEST_RAW_DATA/pose_logs/{log_id}/poses/city_SE3_egovehicle_{timestamp}.json'
+	json_fpath = f'{parent_path}/{log_id}/poses/city_SE3_egovehicle_{timestamp}.json'
 	check_mkdir(str(Path(json_fpath).parent))
 	save_json_dict(json_fpath, pose_dict)
 
+def dump_point_cloud(points, timestamp, log_id, parent_path):
+	data = {'x': points[:,0], 'y': points[:,1], 'z': points[:,2]}
+	cloud = PyntCloud(pd.DataFrame(data))
+	cloud_fpath = f'{parent_path}/{log_id}/lidar/PC_{timestamp}.ply'
+	check_mkdir(str(Path(cloud_fpath).parent))
+	cloud.to_file(cloud_fpath)
+
+def dump_object_labels(labels, timestamp, log_id, parent_path):
+	argoverse_labels = []
+	for label in labels:
+		argoverse_labels.append(build_label(label, timestamp))
+	json_fpath = f'{parent_path}/{log_id}/per_sweep_annotations_amodal/tracked_object_labels_{timestamp}.json'
+	check_mkdir(str(Path(json_fpath).parent))
+	save_json_dict(json_fpath, argoverse_labels)
+
+def build_label(label, timestamp):
+	label_dict = {}
+	label_dict["center"] = {}
+	label_dict["center"]["x"] = label.box.center_x
+	label_dict["center"]["y"] = label.box.center_y 
+	label_dict["center"]["z"] = label.box.center_z
+	label_dict["length"] = label.box.length
+	label_dict["width"] = label.box.width
+	label_dict["height"] = label.box.height
+	label_dict["rotation"] = {}
+	qx,qy,qz,qw = yaw_to_quaternion3d(label.box.heading)
+	label_dict["rotation"]["x"] = qx
+	label_dict["rotation"]["y"] = qy
+	label_dict["rotation"]["z"] = qz
+	label_dict["rotation"]["w"] = qw
+	label_dict["label_class"] = LABEL_TYPES[label.type]
+	label_dict["timestamp"] = timestamp
+	if label.id not in track_id_dict.keys():
+		track_id = uuid.uuid4().hex
+		track_id_dict[label.id] = track_id
```

Comment:
is `track_id_dict` a global variable? I'd prefer it not be

Reply:
Addressed in [this commit](https://github.com/johnwlambert/waymo_to_argoverse/pull/2/commits/8931614c713c462a1efbbc0f06a366d44a28df25)